script/build-windows.ps1

```diff
@@ -29,7 +29,6 @@
 #        .\script\build-windows.ps1
 
 $ErrorActionPreference = "Stop"
-Set-PSDebug -trace 1
```

Comment:
I made this change because it was causing (literally) 9000 lines of unnecessary output in appveyor from the "find .pyc files" script.  Hopefully we can do without it.